src/com/owncloud/android/ui/RadioButtonPreference.java

```diff
@@ -1,39 +0,0 @@
-/**
```

Comment:
This class was included in methods to delete in #2153, I've just moved it to classes to delete and checked it as done.

Reply:
fixed that